app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -293,6 +300,8 @@ class SearchFragment : Fragment(), UserInteractionHandler {
             }
             else -> awesomeBarView.isKeyboardDismissedProgrammatically
         }
+
+        // Note: Actual navigation happens in `handleEditingCancelled` in SearchController
```

Comment:
nit - i like the comment but it looks a little weird at the bottom of this fun? Maybe at the top makes sense?

Reply:
Updated 😄